README.md

```diff
@@ -1,69 +1,77 @@
+bash download_TCGA_data.sh
 ```
 
 ## Analysis
 
+### Recreate manuscript results
+
+After data has been downloaded, running
+
+```
+bash run_all_analyses_and_plots.sh
+```
+
+with v2.0 of this repository will reproduce the results presented in our manuscript.
+
 ### Machine Learning Pipeline
 
 Here's a schematic overview of our machine learning experiments:
 
-![](https://github.com/greenelab/RNAseq_titration_results/blob/master/diagrams/RNA-seq_titration_ML_overview.png)
+![](https://github.com/greenelab/RNAseq_titration_results/blob/master/diagrams/RNA-seq_titration_ML_overview.png?raw=true)
 
 **Overview of supervised and unsupervised machine learning experiments.** 
 
-1. 520 TCGA Breast Cancer samples run on both microarray and RNA-seq were split 
-into a training (2/3) and holdout set (1/3).
-2. RNA-seq’d samples were "titrated" into the training set, 10% at a time (0-100%) 
-resulting in eleven training sets for each normalization method. 
-3. _Machine learning applications._ Three supervised multi-class (BRCA PAM50 subtype) 
-classifiers—LASSO, linear SVM, and Random Forest—were trained on each training set 
-and tested on the microarray and RNA-seq holdout sets. The holdout sets were projected 
-onto and back out of the training set space using two unsupervised techniques, Independent 
-and Principal Components Analysis, to obtain reconstructed holdout sets. The 
-classifiers used in step 4A above were used to predict on the reconstructed holdout 
-sets. 
+1. Matched samples run on both microarray and RNA-seq were split into a training (2/3) and holdout set (1/3).
+2. RNA-seq samples were "titrated" into the training set, 10% at a time (0-100%), replacing their matched array samples, resulting in eleven training sets for each normalization method. 
+3. Machine learning applications
+
+  - _Supervised learning_: 
+Three classifiers - LASSO, linear SVM, and Random Forest — were trained on each training set and tested on the microarray and RNA-seq holdout sets.
+The models were trained to predict tumor subtype (5 classes), and the binary mutation status of _TP53_ and _PIK3CA_.
+
+  - _Unsupervised learning_: 
+Holdout sets were projected onto and back out of the training set space using Principal Components Analysis to obtain reconstructed holdout sets.
+The trained subtype classifiers were used to predict on the reconstructed holdout sets.
+[PLIER](https://github.com/wgmao/PLIER) (pathway-level information extractor) identified active gene expression pathways in each cancer type.
 
 ```
 # To run the machine learning pipeline, run in top directory:
-sh run_machine_learning_experiments.sh
-
-# To run one repeat of the subtype classifier pipeline, use:
-Rscript run_experiments.R
+bash run_machine_learning_experiments.sh [cancer type] [prediction task] [n cores]
 ```
 
+where 
+
+- `[cancer type]` is `BRCA` or `GBM`
+- `[prediction task]` is `subtype`, `TP53`, or `PIK3CA`
+- `[n cores]` is the number of cores you want to run in parallel
+
```

Comment:
Break this and running differential expression analysis into its own section, e.g., "Running individual experiments" below the Methods (or Analysis) section.

Reply:
I think you could tuck the contents of "Other scripts" under the "Running individual experiments" header